jobs/slack-statistics.ts

```diff
@@ -0,0 +1,27 @@
+import { prisma } from "../common/prisma";
+import { sendToSlack, SLACK_CHANNEL } from "../common/slack";
+import { table } from "../common/slack/blocks";
+
+export async function postStatisticsToSlack() {
+    const lastMonthBegin = new Date();
+    lastMonthBegin.setMonth(lastMonthBegin.getMonth() - 1);
+    lastMonthBegin.setDate(1);
+    const begin = lastMonthBegin.toISOString();
+
+    const lastMonthEnd = new Date();
+    lastMonthEnd.setDate(-1);
```

Comment:
If we execute the job manually, it will use the first of last month till yesterday. I would do something similar to the above just to ensure we always select the whole last month.

Reply:
Not quite, actually `setDate` takes the absolute day of the month, but handles over and underflows gracefully, so -1 is indeed the last date of the previous month. However I just noticed that we actually skip one whole day that way, will have to adapt this ...